lib/splitio-api-binding.js

```diff
@@ -53,11 +54,38 @@ class SplitioApiBinding {
     }
   }
 
+  /**
+   * Polls the Split.io API until since and till timestapms are the same.
+   */
+  async getAllChanges ({ path }) {
+    let since = SINCE_VALUE_FOR_FIRST_REQUEST
+    let results = await this.httpGet({ path, since })
+    const allChanges = [results]
+    // The till value represents the timestamp of the last change included in the response.
+    let till = results.till
+    while (since !== till) {
+      since = till
+      results = await this.httpGet({ path, since })
+      till = results.till
+      if (since === till) {
+        return { allChanges, since }
+      }
+      allChanges.push(results)
+    }
+    return { allChanges, since }
+  }
+
   /**
    * Get split data.
    */
-  getSplitChanges () {
-    throw new Error('Not implemented')
+  async getSplitChanges () {
+    const path = '/splitChanges'
+    const splitChanges = []
+    const { allChanges, since } = await this.getAllChanges({ path })
+    allChanges.forEach(change => {
+      splitChanges.push(change.splits)
+    })
+    return { splitChanges, since }
```

Comment:
I wonder why this function returns `since`, will we use it somewhere else?

Reply:
yep, the FE data loader will use it so that the SDK knows what timestamp to use for polling for updates

Reply:
edit: talked to silas more about this -- we'll eventually keep track of this on the backend but are just going to poll with `-1` for now